src/System.Windows.Forms/src/System/Windows/Forms/GroupBox.GroupBoxAccessibleObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class GroupBox
+    {
+        [ComVisible(true)]
+        internal class GroupBoxAccessibleObject : ControlAccessibleObject
+        {
+            internal GroupBoxAccessibleObject(GroupBox owner) : base(owner)
+            {
```

Comment:
Please add check `owner` for null.

Reply:
Maybe we also need to check for IsHandleCreated here. But need to review how `ControlAccessibleObject` deals with this case.